src/ResultWriter/WaveFieldWriter.cpp

```diff
@@ -427,13 +431,7 @@ void seissol::writer::WaveFieldWriter::write(double time)
 				continue;
 			real* managedBuffer = async::Module<WaveFieldWriterExecutor,
 					WaveFieldInitParam, WaveFieldParam>::managedBuffer<real*>(m_variableBufferIds[1] + nextId);
-
-#ifdef _OPENMP
-			#pragma omp parallel for schedule(static)
-#endif // _OPENMP
-			for (unsigned int j = 0; j < m_numLowCells; j++)
-				managedBuffer[j] = m_pstrain[m_map[j]
-						* WaveFieldWriterExecutor::NUM_PLASTICITY_VARIABLES + i];
+			m_lowVariableSubsampler->get(m_pstrain, m_map, i, managedBuffer);
```

Comment:
That doesn't work because pstrain is a low order field and no high order information is available.

Reply:
ha ok so that my problem, thx!

Reply:
would it be difficult to get a high-order field?

Reply:
Not for the stress tensor but for eta you need
eta += timeStepWidth * sqrt(0.5 dstrain_{ij}/dt dstrain_{ij}/dt)
which you'll need to do differently for high order

Reply:
With quadrature, e.g.